src/classes/modules/java.base/jdk/internal/misc/SharedSecrets.java

```diff
@@ -150,7 +152,18 @@ public static JavaObjectInputStreamAccess getJavaObjectInputStreamAccess() {
   public static void setJavaObjectInputStreamAccess(JavaObjectInputStreamAccess access) {
     javaObjectInputStreamAccess = access;
   }
-  
+
+  public static JavaObjectInputFilterAccess getJavaObjectInputFilterAccess() {
+    if (javaObjectInputFilterAccess == null) {
+      unsafe.ensureClassInitialized(ObjectInputFilter.Config.class);
+    }
+    return javaObjectInputFilterAccess;
```

Comment:
Same as with the previous issue: should this code initialize a singleton, or possibly return null, if (javaObjectInputFilterAccess == null)?

Reply:
Our ensureClassInitialized method seems to do nothing. So it will possibly return null.

https://github.com/javapathfinder/jpf-core/blob/09c442cdcdc364aa65fcae08a506b749c47969c5/src/classes/modules/java.base/jdk/internal/misc/Unsafe.java#L215

https://github.com/javapathfinder/jpf-core/blob/09c442cdcdc364aa65fcae08a506b749c47969c5/src/peers/gov/nasa/jpf/vm/JPF_jdk_internal_misc_Unsafe.java#L260-L264

JDK source for ensureClassInitialized:
https://github.com/unofficial-openjdk/openjdk/blob/6caf29475edb2d25cb9349f763451e20aa545c74/src/java.base/share/classes/jdk/internal/misc/Unsafe.java#L1032-L1043

Reply:
I see, this is indeed the same as in the JDK implementation. Merging this one.